doc/changes/latest.inc

```diff
@@ -87,4 +89,4 @@ Bugs
 
 API changes
 ~~~~~~~~~~~
-- Nothing yet
+- Parameter ``alpha`` now defaults to ``'sure'`` in :func:`mne.inverse_sparse.mixed_norm` (:gh:`9430` by **new contributor** `Pierre-Antoine Bannier`_ and `Alex Gramfort`_)
```

Comment:
This does not go under API because there is no deprecation cycle. This needs to go under BUG or we need to add a proper deprecation cycle.

Reply:
before alpha was a required param so working code will not change. There is no change in behavior.
it's clearly an enhancement for me.

Reply:
I would not put this under API changes, then, but just under Enhancements. This section should be reserved for things people need to think about needing to change in their currently working code for it to continue working in the next version